fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -220,6 +220,39 @@ then
 	log_Err_Abort "the --usejacobian option must be 'true' or 'false'"
 fi
 
+
+# ------------------------------------------------------------------------------
+#  Legacy Style Data Options
+# ------------------------------------------------------------------------------
+
+fMRIReference=`opts_GetOpt1 "--fmriref" $@`                          # reference BOLD name run to use for movement correction target and to copy atlas registration from (or NONE; default)
```

Comment:
Unclear to me, is `--fmriref` allowed to be a whole run?  Is it expected to be just a single frame?  If the former, should there be an 'fslroi' operation at some stage to select just a single frame?  (As we've learned, between FSL 5.x and 6.x, the default behavior of some FSL tools changed considerably when using 4D files as inputs, so we had to become more careful about the distinction that we were in the past).

Reply:
`--fmriref` is actually the name of the BOLD that is to be used as the reference. The actual reference is then `$fMRIReference"/"$ScoutName"_gdc`. Which is either the scout for that BOLD or the "generated" scout, which is a mean across the BOLD timeseries after motion correction.